styles/dowhop-profile-avatars.css

```diff
@@ -5,11 +5,10 @@
 .user-avatar-section {
   display: flex;
   flex-wrap: wrap;
+  padding: 2%;
 }
 
 .user-avatar-container {
```

Comment:
Can we delete the CSS selectors that are no longer used?

Reply:
Good call! Thanks for addressing - we should make it a point to clear out unused CSS throughout the app.